packages/twenty-front/src/modules/command-menu/hooks/useCommandMenu.ts

```diff
@@ -216,8 +216,14 @@ export const useCommandMenu = () => {
   );
 
   const onItemClick = useCallback(
-    (onClick?: () => void, to?: string) => {
-      toggleCommandMenu();
+    (
+      shouldCloseCommandMenuOnClick: boolean,
+      onClick?: () => void,
+      to?: string,
```

Comment:
we should use one object parameter for better readability
```suggestion
      {shouldCloseCommandMenuOnClick, onClick, to}: {shouldCloseCommandMenuOnClick: boolean;
      onClick?: () => void;
      to?: string;
      }
```